src/DIRAC/TransformationSystem/Client/WorkflowTasks.py

```diff
@@ -106,11 +103,10 @@ def prepareTransformationTasks(
             owner = proxyInfo["username"]
             ownerGroup = proxyInfo["group"]
 
-        if not ownerDN:
```

Comment:
I am not entirely sure the behavior is the same. As it was before, you could run with a given proxy and still set the `ownerDN` to a different identity and I wonder if we do not rely on that somewhere

Reply:
What you mention is, IIRC, a "trick" used for running the TransformationSystem in a multi-VO setup. I defer the answer to @andresailer

Reply:
I hope this is only used so that tasks are submitted by the owner of the transformation rather than the shifter

Reply:
I believe we still need two ifferent "owner" there. It's actually a confusing naming, because `owner`  and `ownerDN` can be two different identities. It should rather be `owner` and `onBehalfOf` 

It was introduced here https://github.com/DIRACGrid/DIRAC/pull/3723

Reply:
I think that's where it ends up indeed.
It's easy to try though

Reply:
Maybe the solution is to start by storing in TransformationDB an "OnBehalfOf" field that would store the username, and not the DN of the user. But this won't be done in this PR. 
I will see what can be done, but anyway the TS on the DIRAC certification setup is not using the ShifterProxy so it should be possible to test this already.

Reply:
Definitely the TransformationDB should store Author(UserName) instead of AuthorDN.

The Owner, OwnerGroup, OwnerDN should be consistent though. Either all belong to the shifterProxy, or be the author of the transformation.

Reply:
@andresailer could you come up with an integration test for that specific case ?

Reply:
The case where this was designed for, that is for multiVO TS (you introduced it in #3723)